src/systems/thruster/Thruster.cc

```diff
@@ -191,14 +201,21 @@ void Thruster::Configure(
   std::string thrusterTopicOld = ignition::transport::TopicUtils::AsValidTopic(
     "/model/" + ns + "/joint/" + jointName + "/cmd_pos");
 
+  ignwarn << thrusterTopicOld << " topic is deprecated" << std::endl;
+
   this->dataPtr->node.Subscribe(
     thrusterTopicOld,
     &ThrusterPrivateData::OnCmdThrust,
     this->dataPtr.get());
 
   // Subscribe to force commands
-  std::string thrusterTopic = ignition::transport::TopicUtils::AsValidTopic(
-    "/model/" + ns + "/joint/" + jointName + "/cmd_thrust");
+  std::string thrusterTopic =
+    "/model/" + ns + "/joint/" + jointName + "/cmd_thrust";
+
+  if (!this->dataPtr->topic.empty())
+    thrusterTopic = ns + "/" + this->dataPtr->topic;
+
+  thrusterTopic = ignition::transport::TopicUtils::AsValidTopic(thrusterTopic);
```

Comment:
```suggestion
  thrusterTopic = transport::TopicUtils::AsValidTopic(thrusterTopic);
```

Reply:
Removed in [b551759](https://github.com/gazebosim/gz-sim/pull/1681/commits/b5517593ef7453f569c5c0e23f203bb266fee6f1).